CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Change log
-Generated on 2020-11-30
+Generated on 2020-12-11
 
 ## Release 0.3
 
 ### Features
 |||
 |:---|:---|
+|[#444](https://github.com/NVIDIA/spark-rapids/issues/444)|[FEA] Plugable Cache|
+|[#1158](https://github.com/NVIDIA/spark-rapids/issues/1158)|[FEA] Better documentation on type support|
+|[#57](https://github.com/NVIDIA/spark-rapids/issues/57)|[FEA] Support INT96 for parquet reads and writes|
+|[#830](https://github.com/NVIDIA/spark-rapids/issues/830)|[FEA] commonize the version checking in the integration tests|
+|[#817](https://github.com/NVIDIA/spark-rapids/issues/817)|[FEA] Create integration test build against Dataproc|
```

Comment:
Why are features also tagged with test or build put into this section?  I am fine if that is what the requirements ask for.  It just feels like they are not something that end users would look for in the Features section.

Reply:
yeah, thats something we may need to re-adjust the label here, the tool was just generating doc by label and project

![image](https://user-images.githubusercontent.com/8086184/102029294-e2b2ae80-3de8-11eb-8f45-a735cc2e3f01.png)

Reply:
I reviewed the labels on the issues and removed the `feature` label from those issues that were focused on tests.